locales/en.default.schema.json

```diff
@@ -163,7 +163,8 @@
       "name": "Buttons"
     },
     "variant_pills": {
-      "name": "Variant pills"
+      "name": "Variant pills",
+      "paragraph": "Affects the display of product variants on a product page or a featured product section."
```

Comment:
```suggestion
      "paragraph": "Affects product variant pills picker type when selected."
```

Thanks for the ping @katycobb !  
Is the clarity we are aiming for here more for what the setting will do or to define what "Variant pills" are?
I was also wondering if we could go with a shorter form to convey the same thing?

**_At first here were my initial impressions:_**

> Affects how product variant options are displayed when pills are selected as the variant picker type.

⤴️ Descriptive but doesn't tell me what variant pills are.

_**Alternatives if keeping the same intention but shorter + link maybe?**_

> Affects product variant pills picker type when selected. (Learn more)[https://help.shopify.com/en/manual/online-store/themes/theme-structure/theme-settings#variant-pills)

⤴️ Doesn't tell you much more of what Variant pills are but conveys it is an option to be selected somewhere.

_**Alternatives to explain what variant pills are**_

> Variant pills are one of the product variant picker type options.

> Customize the display of the product variant pills picker type when selected. 

⤴️ Too much action driven?

> Affects product variant pills picker type when selected. Variant pills displays the variant options as clickable buttons. 

⤴️ A mix of the 2? I was looking at the [Variant picker block description](https://help.shopify.com/en/manual/online-store/themes/theme-structure/page-types#product-pages#variant-picker-block) and it is [referenced as a button](https://screenshot.click/13-29-nsvay-efj1j.png
). Maybe it was overlooked for consistency?

Reply:
Thank you both for the feedback! 🙌

> "Variant pills are one of the product variant picker type options."

Personally, I kind of like this one — it's straightforward and to the point. Might also be worthwhile to [link to the help page here](https://help.shopify.com/en/manual/online-store/themes/theme-structure/page-types#variant-picker-block)? (Though it has that "button" name caveat that @melissaperreault  mentioned. 😕)

Reply:
@kjellr I'd suggest this slight tweak on the straightforward approach. "Product variant picker type option" might be a bit technical/obscure.

`Variant pills are one way of displaying your product variants. [Learn more](https://help.shopify.com/en/manual/online-store/themes/theme-structure/page-types#variant-picker-block)  `

Re: the help doc, we can ping Kimli from the docs team to make the update :)

Reply:
Thank you! I've updated the PR to reflect the new language + link: 

<img width="283" alt="Screen Shot 2022-06-14 at 3 27 35 PM" src="https://user-images.githubusercontent.com/1202812/173672563-46700e25-1bb8-43e1-a48c-a1113e3470ec.png">

Reply:
cc.  @KimliW Just bringing this to your attention of the usage of `button` inappropriately according to this thread in [this page](https://help.shopify.com/en/manual/online-store/themes/theme-structure/page-types#product-pages#variant-picker-block)